modules/raft/src/integrationTest/java/org/apache/ignite/raft/server/ItJraftCounterServerTest.java

```diff
@@ -193,20 +214,15 @@ protected void after() throws Exception {
             server.stop();
         }
 
-        IgniteUtils.shutdownAndAwaitTermination(executor, 10, TimeUnit.SECONDS);
-
-        TestUtils.assertAllJraftThreadsStopped();
-
-        LOG.info(">>>>>>>>>>>>>>> End test method: {}", testInfo.getTestMethod().orElseThrow().getName());
+        servers.clear();
     }
 
     /**
      * Starts server.
      *
-     * @param idx The index.
-     * @param clo Init closure.
+     * @param idx  The index.
```

Comment:
Our code style doesn't force us to have padding for parameters, as far as I remember

Reply:
It is true.